utils.py

```diff
@@ -977,6 +982,22 @@ def get_latest_folder(folder_path):
     return latest
 
 
+def set_recommended_appimage_config():
+    releases_url = "https://api.github.com/repos/FaithLife-Community/wine-appimages/releases"  # noqa: E501
+    json_data = json.loads(net_get(releases_url))
+    appimage_url = json_data[0].get('assets')[0].get('browser_download_url')
+    logging.info(f"Recommended AppImage URL: {appimage_url}")
+    config.RECOMMENDED_WINE64_APPIMAGE_FULL_URL = appimage_url
+    config.RECOMMENDED_WINE64_APPIMAGE_FULL_FILENAME = os.path.basename(appimage_url)  # noqa: E501
```

Comment:
Could we turn this into a generic function for getting release info from GitHub, such that we could apply it to the wine-appimages repo and the installer repo? This way the function could supply code to both getting the recommended appimage for #5 and also for #46?

`set_recommended_appimage_config()` would then call this generic getter function and set the config to the latest release by running the succeeding parsing code.

Reply:
My thoughts then are that we could work a second function that also sets the config with the latest installer release, too. We could run this function in the background or during start to check if there is a new release for either.